src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+function adderThousandsSeparator(number) {
+  const res = [];
+  const strNum = `${number}`;
+  let count = 0;
+
+  for (let i = strNum.length - 1; i >= 0; i--) {
+    count++;
+    res.push(strNum[i]);
+
+    if (count % 3 === 0 && count !== strNum.length) {
+      res.push(',');
+    }
+  }
+
+  return res.reverse().join('');
+}
```

Comment:
You don`t really need this function. Learn more about [toLocaleString Method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString).